spec/terminology.md

```diff
@@ -3,17 +3,74 @@
+
+[[def: inception event]]
+~ A key establishment event that provides the incepting information needed to derive an [[ref: AID]] and establish its initial [[ref: key state]].
+
+[[def: indirect replay mode]]
+~ An operational mode of the [[ref: KERI]] protocol where the [[ref: KEL]] of an [[ref: AID]] is promulgated to a verifier using witnesses that are disignated by the AID's controller. 
+
+[[def: interaction event]]
+~ A key non-establishment event that anchors external data to the [[ref: key state]] of an [[ref: AID]] as established by the most recent prior establishment event.
+
+[[def: key event]]
+~ A serialized data structure of an entry in the [[ref: key event log]]([[ref: KEL]]) for an [[ref: AID]]. A key event is either establishment or non-establshment. An establishment event changes the [[ref: key state]] of an AID while a non-establishment event does not.
+
 [[def: key event log, KEL]]
-~ As defined by KERI.
+~ A verifiable append-only log of key events for an [[ref: AID]] that is both backward and forward-chained.
 
-[[def: transaction event log, TEL]]
-~ As defined by KERI.
+[[def: key event receipt]]
+~ A message whose body references a [[ref: key event] of an [[ref: AID]] and includes one or more signatures on that key event.
+
+[[def: key event receipt log (KERL), KERL]]
+~ A verifiable append-only log that includes all the consistent [[ref: key event receipt]] messages created by [[ref: witness]]es.
+
+[[def: key event receipt infrastructure (KERI), KERI]]
+~ A protocol that provides an identity system-based secure overlay for the internet. The KERI protocol includes a primary root-of- trust in self-certifying identifiers (SCIDs), called [[ref: autonomic identifier]]s ([[ref: AID]]s), that are strongly bound to an append-only chained [[ref: key event log]] ([[ref: KEL]]) of signed transfer statements provides end verifiable control provenance.
 
 [[def: KERI event stream, KERI event streams]]
 ~ A stream of verifiable KERI data, consisting of the [[ref: key event log]] ([[ref: KEL]]) and other data such as [[ref: transaction event log]] ([[ref: TEL]]). This data is a [[ref: CESR]] event stream (TODO: link to IANA application/cesr media type) and may be serialized in a file using [[ref: CESR]] encoding. We refer to these CESR stream resources as KERI event streams to simplify the vocabulary.
+
+[[def: key state]]
+~ The set of currently authoritative keypairs (current keys) for an [[ref: AID]] and any other information necessary to secure or establish control authority over the AID.
+
+[[def: method-specific identifier, MSI]]
+~ The `method-specific-id` part of DID Syntax, as defined in [DID Core](https://www.w3.org/TR/did-core/#did-syntax). See section [Method-Specific Identifier](#method-specific-identifier).
+
+[[def: out-of-band introduction (OOBI), OOBI]]
```

Comment:
Is a `did:webs` and OOBI, or is there more to an OOBI than gaining access to the Event Log?